backend/api/organization.py

```diff
@@ -12,7 +12,7 @@
 __copyright__ = 'Copyright 2023'
 __license__ = 'MIT'
 
-api = APIRouter(prefix="/api/organizations")
+api = APIRouter(prefix="/api/organization")
```

Comment:
This actually brings up a good point that @KrisJordan mentioned Friday, and something we should clarify. Since we are returning all organizations in this API route, it may be a good idea to keep this API route as `/api/organizations` to imply the expectation that a list should return from this.

Reply:
(Also just a note here so we do not forget, if we do end up with `/api/organizations`, a few API calls in the frontend service may have to be reverted.)